Airdrop.test.js

```diff
@@ -0,0 +1,136 @@
+const {
+    BN,
+    send,
+    constants,
+    expectRevert,
+    assertRevert,
+    balance,
+    ether
+} = require('@openzeppelin/test-helpers');
+require('chai')
+    .use(require('chai-as-promised'))
+    .use(require('chai-bn')(BN))
+    .should();
+
+const Token = artifacts.require("Token");
+const Airdrop = artifacts.require("Airdrop");
+
+contract("Airdrop", ([owner, user1, user2, user3]) => {
+
+    beforeEach(async () => {
+        tokenInstance = await Token.new({ from: owner });
+        airdropInstance = await Airdrop.new(tokenInstance.address, { from: owner });
+    });
+
+    it("Should deposit ether to the contract", async () => {
+
+        await airdropInstance.depositEther({ from: owner, value: ether('2') });
+
+        let etherBalanceAirdrop = await balance.current(airdropInstance.address);
+
+        etherBalanceAirdrop.should.be.bignumber.equal(ether('2'));
+    });
+
+    it("Should update token for airdrop", async () => {
+
+        let newToken = await Token.new({ from: owner });
+        await airdropInstance.updateTokenAddress(newToken.address);
+
+        let actualTokenAddress = await airdropInstance.getTokenAddress();
+
+        actualTokenAddress.should.be.equal(newToken.address);
+    });
+
+    describe("drop ether", async () => {
+

```

Comment:
Add test: Should be fail if array is empty
Add test: Should be fail if amount Is zero